ajax/libs/workbox/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "workbox",
+  "version": "3.0.0",
+  "filename": "workbox-sw.js",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "This module makes it easy to get started with the Workbox service worker libraries.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/workbox"
+  },
+  "keywords": [
+    "workbox",
+    "workboxjs",
+    "service worker",
+    "sw"
+  ],
+  "npmName": "workbox-sw",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/workbox-sw*"
```

Comment:
Let's use exclusive pattern. Notice there are some files we need or don't need in the old versions. Try to figure it out on your own or ask us or opening an issue to check.

Reply:
I change the pattern, but it still exists some files in `test` folder for versions that I can't exclude it.